packages/lmi/src/lmi/llms.py

```diff
@@ -753,7 +753,7 @@ async def acompletion_iter(
                 used_model = completion.model or self.name
             choice = completion.choices[0]
             delta = choice.delta
-            if hasattr(choice.logprobs, "content"):
+            if getattr(choice.logprobs, "content", None):
```

Comment:
Can you add a comment saying that sometimes `"content"` is not an attribute, and sometimes it's `NoneType`